scripts/install_environment.sh

```diff
@@ -87,6 +89,19 @@ function is_libvirtd_tcp_socket_enabled_and_running() {
     fi
 }
 
+function allow_libvirt_cross_network_traffic() {
+    # Flush LIBVIRT_FWI chain each time the network configuration is being updated
+    echo "Installing libvirt network hook to allow cross network traffic"
+    hook_dir="/etc/libvirt/hooks/network.d"
+    hook_filename="${hook_dir}/allow-cross-network-traffic.sh"
+    sudo mkdir -p "${hook_dir}"
+    sudo tee "${hook_filename}" << EOF
+#!/usr/bin/env sh
+iptables -F LIBVIRT_FWI || true
```

Comment:
Long form preferred in scripts

```suggestion
iptables --flush LIBVIRT_FWI || true
```